terraform/digitalocean/k8s/variables.tf

```
@@ -14,7 +8,7 @@ variable "region" {
   description = "(Required) Provide DigitalOcean region"
 }
 
-variable "version" {
+variable "k8s_version" {
   type        = string
   description = "Provide DigitalOcean Kubernetes minor version (e.g. '1.16' or '1.15')"
   default     = "1.16"
```

You specify `1.17` in `.cluster.dev/digitalocean-k8s.yaml`
```suggestion
  description = "Provide DigitalOcean Kubernetes minor version (e.g. '1.15' or higher)"
  default     = "1.17"
```